superset-frontend/src/SqlLab/components/SouthPane/index.tsx

```diff
@@ -152,7 +152,7 @@ export default function SouthPane({
             query={latestQuery}
             actions={actions}
             user={user}
-            height={innerTabContentHeight}
+            height={innerTabContentHeight + 24}
```

Comment:
could we add this to a constant at the top of the file? and explain in code why this is needed?

Alternatively, where does innerTabContentHeight come from? Is there some alteration that needs to be made there instead?

Reply:
I did some inveestigation about `innerTabContentHeight`, but no way to handle it dynamically. 
I will add constant value at the top of the file. 👍